src/MatLang.jl

```diff
@@ -1,20 +1,76 @@
 module MatLang
 using Requires
+using LinearAlgebra
 
-export clcM
+# Main Functions
+export clcM, eyeM, diagM
+
+# Native Julia functions that are just a copy:
+export zerosM, onesM, randM
+
+
+# Command Window (REPL console):
 ################################################################
 """
     clcM
 
 Clear Command Window for Juno or Atom IDE
 """
 function clcM()
-#    Juno.clearconsole();
+#    Juno.clearconsole(); # defined in @require instead
 end
 ################################################################
 
 
 ################################################################
+
+# Matrices and Arrays:
+################################################################
+const zerosM = zeros
+################################################################
+const onesM = ones
+################################################################
+const randM = rand
+################################################################
+"""
+    eyeM(s1,s2,...)
+    eyeM(sizeAsTuple)
```

Comment:
Although it's not efficient, we can add a permanent depwarn on sizeAsArray, since in Matlab we express size as `[2 3]`

Reply:
Yes, this is exactly I said:

> I will create another method that converts a matrix to a tuple, since, In Matlab, we can pass size as a matrix (instead of tuple).

Reply:
But the current implementation doesn't throw any warning. you can use `@deprecate` or `Base.depwarn` to achieve this.

Reply:
I don't think bugging the user is a good idea. I still have the warning in the documentation.